### PR TITLE
Added the method of finding slope from a graph

### PR DESCRIPTION
**Pull request by thisisamank**
- [x] I have read [freeCodeCamp's contribution guidelines](https://github.com/freeCodeCamp/freeCodeCamp/blob/master/CONTRIBUTING.md).
- [x] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [x] My pull request targets the `master` branch of freeCodeCamp.
- [x] None of my changes are plagiarized from another source without proper attribution.
- [x] My article does not contain shortened URLs or affiliate links.

If your pull request closes a GitHub issue, replace the XXXXX below with the issue number.

Closes #XXXXX

### CODE REVIEW

File: guide/english/mathematics/finding-slope-from-graph/index.md [hunk 1]
```diff
@@ -3,13 +3,23 @@ title: Finding Slope from Graph
...
-This is a stub. <a href='https://github.com/freecodecamp/guides/tree/master/src/pages/mathematics/finding-slope-from-graph/index.md' target='_blank' rel='nofollow'>Help our community expand it</a>.
 
-<a href='https://github.com/freecodecamp/guides/blob/master/README.md' target='_blank' rel='nofollow'>This quick style guide will help ensure your pull request gets accepted</a>.
 
-<!-- The article goes here, in GitHub-flavored Markdown. Feel free to add YouTube videos, images, and CodePen/JSBin embeds  -->
+let us consider the graph for 2x+3
 
-#### More Information:
-<!-- Please add any articles you think might be helpful to read before writing the article -->
+https://graphsketch.com/?eqn1_color=1&eqn1_eqn=2x%2B3&eqn2_color=2&eqn2_eqn=&eqn3_color=3&eqn3_eqn=&eqn4_color=4&eqn4_eqn=&eqn5_color=5&eqn5_eqn=&eqn6_color=6&eqn6_eqn=&x_min=-17&x_max=17&y_min=-10.5&y_max=10.5&x_tick=1&y_tick=1&x_label_freq=5&y_label_freq=5&do_grid=0&do_grid=1&bold_labeled_lines=0&bold_labeled_lines=1&line_width=4&image_w=850&image_h=525
```
Comment (nikrb) [inline guide/english/mathematics/finding-slope-from-graph/index.md]: @thisisamank the link isn't displaying properly, can you fix it?

**Review by nikrb — DISMISSED** [review]
We'll be happy to accept this pr if you could just fix the link display

**Review by Nirajn2311 — DISMISSED** [review]
I think that it will be better if you could add a graph for the equation.
Also could you correctly punctuate the sentences.

**Comment by thisisamank** [comment]
Hi thanks for reply...
Can you pls tell me a way of doing it...

**Pull request merged**